src/wolfgame.py

```diff
@@ -72,62 +74,65 @@
     get_target, change_role, match_role, match_mode
    )
 
+if TYPE_CHECKING:
+    from src.channels import Channel
+
 # done this way so that events is accessible in !eval (useful for debugging)
 Event = events.Event
 EventListener = events.EventListener
 
 # Game Logic Begins:
 
-var.LAST_STATS = None
-var.LAST_ADMINS = None
-var.LAST_GSTATS = None
-var.LAST_PSTATS = None
-var.LAST_RSTATS = None
-var.LAST_TIME = None
-var.LAST_GOAT = UserDict() # type: UserDict[users.User, datetime]
+var.LAST_STATS = None  # type: ignore
```

Comment:
omit all ignores. If mypy is throwing an error for these lines, suppress the individual error instead of opting out of the type system entirely. If you don't know the type, omit the annotation. Otherwise fill in the correct type.